.github/workflows/run-test-harness.yml

```diff
@@ -23,4 +23,6 @@ jobs:
           propagate_failure: true
           trigger_workflow: true
           wait_workflow: true
-          client_payload: '{ "sha": "${{ github.event.pull_request.head.sha }}", "sdk": "dotnet" }'
+          client_payload: '{ "sha": "${{ github.event.pull_request.head.sha }}", "sdk": "dotnet", "title": "${{ github.event.pull_request.title }}"}'
```

Comment:
I might add an additional specifier to the title that tells us where the PR is from, something like :
"DotNet PR: ${{ github.event.pull_request.title }}"

Reply:
done!